adapta/storage/blob/README.md

```diff
@@ -128,13 +160,45 @@ print(list(blobs))
 
 
 # Downloads blobs from the S3 storage path to the provided local path, as if you were navigating within the S3 path.
-s3_client.download_blobs(s3_path, local_path="/tmp/path/to/download")
+s3_client.download_blobs(s3_folder_path, local_path="/tmp/path/to/download")
 
 # Upon executing the command 'ls /tmp/path/to/download' in your terminal, a list of files be visible:   
 # 0-c309720b-3577-4211-b403-bbb55083e5c3-0.parquet  _delta_log
+```
+
+
+#### Copy a single blob to a different location:
+```python
+import os
+from adapta.security.clients import AwsClient
+from adapta.security.clients.aws._aws_credentials import EnvironmentAwsCredentials
+from adapta.storage.models.aws import S3Path
+from adapta.storage.blob.s3_storage_client import S3StorageClient
+
+# Set up environment variables
+os.environ["PROTEUS__AWS_ACCESS_KEY_ID"] = <aws_access_key_id>
+os.environ["PROTEUS__AWS_SESSION_TOKEN"] = <aws_session_token>
+os.environ["PROTEUS__AWS_SECRET_ACCESS_KEY"] = <aws_secret_access_key>
+os.environ["PROTEUS__AWS_REGION"] = "eu-central-1"
+os.environ["PROTEUS__AWS_ENDPOINT"] = "http://example.com"
+
+# Create client
+credentials = EnvironmentAwsCredentials()
+aws_client = AwsClient(credentials)
+
+# Initialize storage client
+s3_client = S3StorageClient.create(auth=aws_client)
 
 # Copy blob from one location to another in S3
-target_blob_path='s3a://path/to/blob_copy/'
-s3_target_blob_path = S3Path.from_hdfs_path(target_blob_path)
-s3_client.copy_blob(blob_path=s3_path, target_blob_path=s3_target_blob_path, doze_period_ms=1000) # Time in ms between files being copied
+s3_target_blob_path = S3Path.from_hdfs_path('s3a://bucket/path_to_blob_copy')
+s3_client.copy_blob(blob_path=s3_path, target_blob_path=s3_target_blob_path)
+
+# List all blob files in the target folder
+blob_list = s3_client.list_blobs(blob_path=s3_target_blob_path)
+for blob_details in blob_list:
+    print(blob_details)
+
+#  The list of blobs and their details will be printed out, mirroring the attributes of the original files::
```

Comment:
Same here?

Reply:
Updated in [c2d3fb0](https://github.com/SneaksAndData/adapta/pull/447/commits/c2d3fb0061e54db65a13b1bc094c1889e923e0ef)